container/doctabs.go

```diff
@@ -244,7 +244,12 @@ func (r *docTabsRenderer) Layout(size fyne.Size) {
 	r.updateCreateTab()
 	r.updateTabs()
 	r.layout(r.docTabs, size)
+
+	// lay out buttons before updating indicator, which is relative to their position
+	buttons := r.scroller.Content.(*fyne.Container)
+	buttons.Layout.Layout(buttons.Objects, buttons.Size())
```

Comment:
Is it not possible to just do `r.scroller.Content.Resize(r.scroller.Content.Size())` or something along those lines?
Looking at the container code, it seems to me like it does the same thing without casting.

Reply:
Maybe something like this would work? Untested:
```suggestion
	buttons := r.scroller.Content
	buttons.Resize(buttons.Size())
```

Reply:
I don't think so - when you resize something to the same size it was before the layout will not be called.

Reply:
Alright